lvm_read.py

```diff
@@ -42,6 +42,21 @@ def _lvm_dump(lvm_data, filename, protocol=-1):
     pickle.dump(lvm_data, output, protocol=protocol)
     output.close()
 
+def get_separator(file):
+    separators = {'Tab': '\t',
+                  'Comma': ','
+    }
+    for line in file:
+        if line.startswith('Separator'):
+            separator = line.strip()[10:]
+            break
+    file.seek(0) 
+    if separator in separators.keys():
+        return separators[separator]
```

Comment:
From the specification: "To find out what the separator character(s) is, read the entire header block and search for the keyword Separator. The character(s) that follows the keyword is the separator." Any separator can be supported by just using the character following 'Separator'.